docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -1092,18 +1092,42 @@ When you are ready to add another application to the repo, you'll probably want
 
 You can also go through the full [Angular monorepo tutorial](/getting-started/tutorials/angular-monorepo-tutorial)
 
+## CI with Nx
+
+This tutorial walked you through how Nx can improve the developer experience for local development, but Nx can also make a big difference in CI. Without adequate tooling, CI times tend to grow exponentially with the size of the codebase. Nx helps reduce wasted time in CI with the [`affected` command](/ci/features/affected) and Nx Replay's [remote caching](/ci/features/remote-cache). Nx also [efficiently parallelizes tasks across machines](/ci/concepts/parallelization-distribution) with Nx Agents. You can follow a full tutorial on using Nx in [GitHub Actions](/ci/intro/tutorials/github-actions) or [Circle CI](/ci/intro/tutorials/circle).
```

Comment:
> You can follow a full tutorial on using Nx in [GitHub Actions](/ci/intro/tutorials/github-actions) or [Circle CI](/ci/intro/tutorials/circle).

I'd move this sentence to the end of the paragraph. Ppl tend to skip over this so they'd find this sentence at the end to dive deeper into a dedicated CI tutorial

Reply:
I assume you mean move it to the end of the section, since it is already at the end of the paragraph.  If I moved it to the end of the section, then it would be right next to the identical links that are the first links in the `Next Steps` section.

I'm happy to make these links more prominent somehow, but I don't think what you're asking for makes sense.

Reply:
yeah I meant move to the end of the section.. just saw you also have it as "Next steps". Isn't that kind a duplication. Like if you have a dedicated section about CI, I'd expect that section to have the links clearly listed. And rather not have it again in the Next steps.

Reply:
got it, I'll move the CI links into the CI section